### DISCUSSION TITLE
Astroid Quickstart Cracked Open 201 (Video Tutorial) Questions...

### DISCUSSION THREAD

**Discussion by paurray** [templaza/astroid-framework#589] [Q&A] [answered]
Hello Astroid Community,

working on part one of a series with working title "Astroid Quickstart Cracked Open 201"

Am working on a script and 2 sections in on the first part...

- Menus & Templates
- Modules & Menus

I get pretty much every thing so far. Just not what is going on with the Astroid Menu set to the position "Status" is this just parked here for possible use in the future or is it actually being used on the site some where?

hope this makes sense

thanks

Paul

<img width="960" alt="WhatIsTheStatusThing" src="https://github.com/templaza/astroid-framework/assets/9163424/c2b39638-5724-47dc-bac4-5848270c5da0">

**Comment by paurray** [comment]
This is the first of several videos that will be combined into a playlist dealing with the Astroid Framework for Joomla. Specifically the Quickstart package which includes the Boilerplate Template. Two! Article follows…. 

https://www.youtube.com/watch?v=KNWwmstUp7A

Please, like, share and comment :-)

**Comment by paurray** [comment]
Article is up here:
https://astroidframe.work/blog/astroid-hands-on/deep-dive-astroid-boilerplate-site-201

@sonvnn @Chacapamac 

> I get pretty much every thing so far. Just not what is going on with the Astroid Menu set to the position "Status" is this just parked here for possible use in the future or is it actually being used on the site some where?

Could either of you or someone else confirm if my assumption is correct? Would like to address thins amongst other things in "Deep Dive Astroid Boilerplate Site 202" :-)

**Comment by Chacapamac** [comment, marked as answer]
@paurray — Great clear Video to show people the base in setting the menu 
— **Great Job! Mr. Dust** —  _( I guess it’s Star dust ✨...)_ 😀
